app/assets/stylesheets/notifications.scss

```diff
@@ -1,9 +1,12 @@
 @import 'variables';
 @import 'mixins';
 @import 'functions';
+@import 'config/import';
 
 .notifications-index {
   .home {
+    overflow: initial;
```

Comment:
Could you help me understand what does it do?

Reply:
I'm not sure why, but without this line, the `.home` container was hiding a portion of the menu that comes up. I noticed that in another file, `.home` has `overflow: hidden` as the default.

Reply:
I've removed the `overflow: hidden` in my PR for the comments subscription because I had the same issue. (PR is still in Draft mode though). Unless there is some use case that we need to hide the overflow, I think it's totally fine to remove `overflow: hidden` from _top.html.erb. Maybe @benhalpern had a reason for this at some point. I see it was added in https://github.com/thepracticaldev/dev.to/commit/630c1b7425ed38b187f15e21415708145472fd46